packages/transaction-controller/src/TransactionController.ts

```diff
@@ -250,7 +254,8 @@ export class TransactionController extends BaseController<
    * @param options - The controller options.
    * @param options.blockTracker - The block tracker used to poll for new blocks data.
    * @param options.disableHistory - Whether to disable storing history in transaction metadata.
-   * @param options.disableSendFlowHistory - Explicitly disable transaction metadata history.
+   * @param options.disableSendFlowHistory - Whether to disable transaction metadata history.
+   * @param options.disableSwaps - Whether to disable additional processing swaps transactions.
```

Comment:
```suggestion
   * @param options.disableSwaps - Whether to disable additional processing on swaps transactions.
```

Reply:
My fault, sorry Goktug, missed a word.

Reply:
Fixed in https://github.com/MetaMask/core/pull/1877/commits/0af0ae40051eca3c12741b0942d37a9fe878a96b